dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1492,19 +1497,29 @@ def median_abs_deviation(self) -> float:
             np.append([True], np.diff(bin_edges_impose) > 1e-14)
         ]
 
-        bin_counts_impose_pos: float = np.interp(
-            bin_edges_impose, bin_edges_pos, np.cumsum(np.append([0], bin_counts_pos))
+        bin_counts_impose_pos: float = cast(
```

Comment:
I did a check on this. These are not actually float. That's part of the reason this is erroring and causing bugs elsewhere. This is actually returning an NDArray of np.float64

Reply:
Done.

Reply:
great